### PR TITLE
climate-sensor enhancement

### LINKED ISSUE #75: Climate sensor should output which sensor it's from AND separate outputs for temp / humidity would be a nice option
Normally you can derive from msg.topic which device  it's from. 
Because Climate Sensor keeps this a secret I have to add the sensor-name in a topic for each sensor, before I forward it to a graph.
Please add the name of the climate sensor to msg.topic so I can forward the message directly to a graph.

Also, in addition, it would be nice if it had the option of multiple outputs so I can connect the Humidity output to a humidity-graph, temp to temp, etcetera.

### PR DESCRIPTION
**Pull request by Dirnei**
- Added `deviceName` to the ouput of the climate-sensor. Resolves #75
- Added `separateOuputs` option the the climate sensor. Resolves #75

### CODE REVIEW

File: src/nodes/sensors.html [hunk 1]
```diff
@@ -163,10 +214,18 @@ <h3>Output</h3>
...
-
+        <div class="form-row">
+            <label for="node-input-separateOutputs"><i class="fa fa-arrows-alt"></i> Separate outputs</label>
+            <input type="checkbox" id="node-input-separateOutputs" />
+        </div>
     </div>
 </script>
 
 <script type="text/html" data-help-name="climate-sensor">
-    <p>Node to trigger scenes</p>
```
Comment (Andreas-Huber) [inline src/nodes/sensors.html:171]: I started to add some docs there. But then I realized that I don't know the outputs and I don't have a sensor to test it, nor the time now. Could you maybe document those quickly?

**Review by Andreas-Huber — CHANGES_REQUESTED** [review]

File: docs/nodes/climate-sensor.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# Climate sensor
```
Comment (Andreas-Huber) [inline docs/nodes/climate-sensor.md:1]: Nice! More docs :-)

**Comment by Andreas-Huber** [comment]
I tested setting the output config. Seems to work. Just don't have a sensor to test the sending as well.

**Review by Andreas-Huber — APPROVED** [review]

File: src/nodes/sensors.html (hunk 1 above)
Comment (Dirnei) [reply src/nodes/sensors.html:171]: Added more description :) I completely forgot about the data-help ^^

**Pull request merged**